app/qml/AuthPanel.qml

```diff
@@ -46,17 +46,18 @@ Item {
     registrationForm.clean()
   }
 
+  id: root
+  focus: true
+
   Keys.onReleased: {
+
     if (event.key === Qt.Key_Back || event.key === Qt.Key_Escape) {
-      event.accepted = true
+      event.accepted = true;
```

Comment:
why is ; needed if there is newline?

Reply:
not needed, I was just use to place it even in `js`